prow/cmd/crier/main.go

```diff
@@ -78,21 +87,40 @@ func (o *options) Validate() error {
 		}
 	}
 
+	if o.githubWorkers > 0 {
+		for _, group := range []flagutil.OptionGroup{&o.github} {
+			// no dry-run mode for crier
+			if err := group.Validate(false); err != nil {
+				return err
+			}
+		}
+	}
+
 	return nil
 }
 
 func gatherOptions() options {
 	o := options{
 		gerritProjects: gerritclient.ProjectsFlag{},
 	}
+	fs := flag.NewFlagSet(os.Args[0], flag.ExitOnError)
```

Comment:
note: Didn't we recently refactor flags into some common library. Should we consider using this?

Reply:
I copied from whatever latest is from other controllers :joy: did we change something else? reference?

Reply:
I wouldn't worry about it